README.md

```diff
@@ -55,13 +57,33 @@ Any merged PRs with an **Editor Review** or **Reviews Complete** label will be p
 
 ## Building and Validating
 
-If you've downloaded the repo and are editing Mattermost documentation on your local machine, you can generate the HTML files from markdown in the `/source` directory. You can review them before you commit changes or create pull requests. 
-
-1. Download the Mattermost docs repo onto a machine with Python installed.
-2. Install [pipenv](https://docs.pipenv.org/): `pip install pipenv`. 
-3. `cd` into the cloned repository.
-4. Install the required packages: `pipenv install`.
+If you've downloaded the repo and are editing Mattermost documentation on your local machine, you can generate the HTML files from markdown in the `/source` directory. You can review them before you commit changes or create pull requests.
+
+**Note:** Commands can be executed on Linux, Mac, and Windows (using Powershell)
+
+1. Open a terminal window and clone the forked copy of the docs repo by running:
+```sh
+$ git clone https://github.com/mattermost/docs.git
```

Comment:
Including $ characters at the beginning of CLI entries may suggest to the reader that the $ is required (when it's not). I recommend removing these instances for clarity.

Reply:
I think whever we open a tereminal we have these symbols that could help people understand that we need to run exactly these commands. 
Not sure if we need to remove these.

Reply:
@justinegeffen - I'd like your perspective on this.

Reply:
We don't specify this in the style guide so I am 0/5 on which we want to go with provided we pick one and then apply it consistently.

Reply:
If our goal is to provide command line entries that can be easily copied and pasted directly from docs to a terminal, I recommend that we remove them to avoid cases where the coped commands include the command prompt symbol. 

Further, because the command prompt symbol is often configurable, different users may see different symbols than what is documented. I'm 3/5 for removing the symbols and including this rationale in the Style Guide for omitting them in documented commands.

Reply:
Hi @cwarnermm and @justinegeffen 
I think we are then moving forward with removing the `$` symbol. Thanks for the feedbacks.